README.md

```diff
@@ -32,6 +32,21 @@ external evaluation engine in a vendor agnostic way.
 
 The OpenFeature project will include client libraries for common technology stacks including, but not limited to:
 
-* Golang
-* Java
-* JavaScript/TypeScript
+- Golang
+- Java
+- JavaScript/TypeScript
+
+### Specification style guide and tooling
+
+This specification complies with [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) and seeks to conform to the [W3C QA Framework Guildlines](https://www.w3.org/TR/qaframe-spec/).
+
+In accordance with this, some basic tooling (donated graciously by [Diego Hurtado](https://github.com/ocelotl)) has been employed to parse the specification and output a JSON structure of concise requirements, highlighting the particular `RFC 2119` verb in question.
+
+To parse the specification, simply type `make`. Please review the generated JSON files, which will appear as siblings to any of the markdown files in the `/specification` folder.
+
+- Use code blocks for examples.
```

Comment:
I would add "each sentence on its own line" which makes it MUCH easier to review in pull request form because I can make suggestions, comments, and corrections on a single sentence rather than a multi-sentence line.

Reply:
That's and interesting suggestion.

Reply:
I added this to the style guide, an opened: https://github.com/open-feature/spec/issues/36